internal/repotesting/repotesting.go

```diff
@@ -70,7 +76,14 @@ func (e *Environment) setup(tb testing.TB, version content.FormatVersion, opts .
 		}
 	}
 
-	st := blobtesting.NewMapStorage(blobtesting.DataMap{}, nil, openOpt.TimeNowFunc)
+	var st blob.Storage
+	if opt.RetentionPeriod == 0 && opt.RetentionMode == "" {
```

Comment:
might not be too important or relevant, but just wondering if this should be `||` instead of `&&`, to match the behavior of `retentionBlob`'s `IsNull` method?

Reply:
Yeah, will make it consistent with IsNull.